### PR TITLE
refactor: use php 7.1 features

### PR DESCRIPTION
**Pull request by Niktux**
ping @lebris

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ONYX_DIR=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 USER_ID=$(shell id -u)
 GROUP_ID=$(shell id -g)
+COMPOSER_ARGS=--ignore-platform-reqs
```
Comment (Niktux) [inline Makefile]: That's already the case : https://github.com/Onyx-org/skeleton/blob/php71/composer.json#L8

File: README.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+## Overview
+
+PHP application skeleton based on Silex 2. Promote framework agnostic conception.
+
+## Getting started
+
+### Requirements
+
+* docker, executable by your user (see official documentation to set it up)
+* php >= 5.6
```
Comment (Niktux) [inline README.md]: Typo

File: Makefile (hunk 1 above)
Comment (lebris) [inline Makefile]: Nope. 
That's a requirement, not a platform definition.

Comment (Niktux) [inline Makefile]: That's right but does not love this configuration entry. We can not assume the current version of php used by each project bootstrapped with Onyx. We can only enforce requirements.

**Pull request merged**